src/tests/test_api.rs

```diff
@@ -25,6 +27,7 @@ pub trait TestApi {
     async fn create_index(&mut self, table: &str, columns: &str) -> crate::Result<String>;
 
     fn system(&self) -> &'static str;
+    fn connector_tag(&self) -> Tags;
```

Comment:
Last thing: unfortunately, `JSON_CONTAINS` doesn't behave the same on MariaDB as it does on MySQL. This was causing issues on test assertions. I, therefore, added a new method to enable getting the current tag of the connector. See the comment below how it's being used 👇🏻